acceptance/README.md

```diff
@@ -1,194 +1,250 @@
 ```
-BEAKER_KEYFILE=~/.ssh/id_rsa-mykey          \
-GIT_SHA=309e197                             \
-BOLT_CONTROLLER=centos7-64                  \
-BOLT_NODES=ubuntu1604-64,windows10ent-64    \
-SSH_PASSWORD='S3@ret3'                      \
-WINRM_PASSWORD='S3@ret3'                    \
+GIT_SHA=309e197  \
+SSH_PASSWORD='S3@ret3' \
+WINRM_PASSWORD='S3@ret3' \
 bundle exec rake test:git
 ```
 
 Example to test topic branch on fork of bolt on GitHub
 ```
-GIT_FORK=octocat/bolt.git                   \
-GIT_BRANCH=my-topic-branch                  \
-BEAKER_KEYFILE=~/.ssh/id_rsa-mykey          \
-BOLT_CONTROLLER=centos7-64                  \
-BOLT_NODES=ubuntu1604-64,windows10ent-64    \
-SSH_PASSWORD='S3@ret3'                      \
-WINRM_PASSWORD='S3@ret3'                    \
+GIT_FORK=octocat/bolt.git          \
+GIT_BRANCH=my-topic-branch         \
+SSH_PASSWORD='S3@ret3'           \
+WINRM_PASSWORD='S3@ret3'          \
 bundle exec rake test:git
 ```
 
 #### Package
 Example to test a specific package version from Puppet's internal package repos
 ```
 SHA=261d55b5cc1e8a6d00f4ff4573e17048bea08c10 \
-BEAKER_KEYFILE=~/.ssh/id_rsa-mykey           \
-BOLT_CONTROLLER=centos7-64                   \
-BOLT_NODES=ubuntu1604-64,windows10ent-64     \
-SSH_PASSWORD='S3@ret3'                       \
-WINRM_PASSWORD='S3@ret3'                     \
+SSH_PASSWORD='S3@ret3' \
+WINRM_PASSWORD='S3@ret3' \
 bundle exec rake test:package
 ```
 
 ### Beaker
-The above rake tasks are the simplest method of executing the acceptance tests
-for `bolt`. However, they assume that the default output of
+
+The above rake tasks are the simplest method of executing the acceptance
+tests for `bolt`. However, they assume that the default output of
 `beaker-hostgenerator` will meet your needs. If this is not the case, you can
 use `beaker-hostgenerator` to create the template for your beaker hosts file,
-modify it to suite your infrastructure, and run beaker independently. This can
-be helpful if you have existing hosts or do not want to use the `vmpooler`
-hypervisor.
+modify it to suite your infrastructure, and run beaker independently. This
+can be helpful if you have existing hosts or do not want to use the
+`vmpooler` hypervisor.
 
-####  Hosts file
+#### Hosts file
 
-You should run `beaker-hostgenerator` to create the hosts.yaml file used by
+You should run `beaker-hostgenerator` to create the `hosts.yaml` file used by
 beaker to run the tests. The examples below illustrate common platforms to
 test against.
 _Centos 7_
 The following example generates a host file with a Centos 7 bolt controller
 with other operating systems used as external nodes to be acted upon by
 `bolt`.
 ```
- bundle exec beaker-hostgenerator centos7-64bolt,ssh.-ubuntu1604-64ssh.-osx1012-64ssh.-windows10ent-64winrm. > hosts.yaml
+A bundle exec beaker-hostgenerator centos7-64bolt,ssh.-ubuntu1604-64ssh.-osx1012-64ssh.-windows10ent-64winrm. > hosts.yaml
```

Comment:
This change doesn't make sense, previously it was a command you could run. It's in a code block.